packages/FlexGrid/FlexGrid.jsx

```diff
@@ -17,6 +18,13 @@ import styles from './FlexGrid.modules.scss'
 const FlexGrid = ({ centre, limitWidth, gutter, children, ...rest }) => {
   const gutterStyle = gutter ? undefined : styles.gutterless
 
+  if (centre) {
+    deprecate(
+      'core-flex-grid',
+      'The centre prop is deprecated due to the limitWidth prop centring the grid on its own. Please remove the centre prop from your grid definition.'
+    )
+  }
+
```

Comment:
There are no tests for this behaviour/code path. Please add a test that asserts that the `deprecate` function is called when using the `centre` prop.

[See a similar test here](https://github.com/telusdigital/tds/blob/master/packages/Image/__tests__/Image.spec.jsx#L38-L41).